core/scss/utilities/mixins/layout/_sticky-footer.scss

```diff
@@ -0,0 +1,27 @@
+@charset 'UTF-8';
+
+//
+// @sticky-footer
+//
+// Slam the footer to the bottom of the window below all content no matter
+// the size of the browser or content.
+//
+// Apply this mixin to the body tag only. It will set the body tag height to
+// 100% and set it to display as #grid. This mixin will then target a direct
+// decendant element with the class `.global-footer` and set it to css grid's
+// align-self: flex-end; This will keep the footer at the bottom of the page
+// when the content is shorter than the window but all the content to push the
+// footer below the viewport when the page is long.
+//
+// Style guide: Mixins.Layout.Sticky Footer
+//
+@mixin sticky-footer {
+  display: grid;
+  height: 100vh; // Avoid the IE 10-11 `min-height` bug.
+  @include padding(0);
+  @include margin(0);
+
+  > .global-footer {
+    align-self: flex-end;
+  }
+}
```

Comment:
I think we should pass the selector for the footer as an argument to the mixin, rather than hardcoding it. On all of UComm's sites, including homesite, "global-footer" is just the universal footer; all our sites also have a fat footer above the universal footer, and both are wrapped in the `<footer>` element. We would want to replace `> .global-footer` with `> <footer>`. But I can see that wouldn't work for everyone.

Reply:
I'd be ok with that. Keep the current model as the default but provide override.